nerfstudio/data/pixel_samplers.py

```diff
@@ -282,16 +321,20 @@ def sample_method(
             # Note: if there is a mask, sampling reduces back to uniform sampling
             indices = super().sample_method(batch_size, num_images, image_height, image_width, mask=mask, device=device)
```

Comment:
I agree, I changed it to `raise NotImplementedError()` but that was the behavior before so this would mess up anyone using masks with the `PatchPixelSampler`

Reply:
True, but I think this is okay. They should switch to using a non-patch sampler if using a mask.